packages/ata/src/index.ts

```diff
@@ -29,9 +29,22 @@ export interface ATABootstrapConfig {
   fetcher?: typeof fetch
   /** If you need a custom logger instead of the console global */
   logger?: Logger
+  /** Wheather to use package.json as the source of truth for transitive dependencies' versions */
+  usePackageJson?: boolean
```

Comment:
Maybe a name like `resolveDependenciesFromPackageJson` is a bit more descriptive, although this is actually the strategy to resolve dependencies _of dependencies_.